### PR TITLE
Update PHP SDK to 2.8.3

### PR DESCRIPTION
**Pull request by sonyy-m**

### CODE REVIEW

**Review by sofiya02 — APPROVED** [review]
LGTM

File: includes/rzp-payment-buttons.php [hunk 1]
```diff
@@ -141,58 +137,57 @@ function column_title($item)
...
-
         $count = count($payment_page);
-        for($i=0;$i<$count;$i++){
-            
-            if($i >= $offset && $i < $offset+$per_page){
+        $payment_pages = array();
+
+        for($i=0;$i<$count;$i++)
+        {
+            if($i >= $offset && $i < $offset+$per_page)
```
Comment (sofiya02) [inline includes/rzp-payment-buttons.php:169]: instead of && replace with and

File: includes/rzp-subscription-buttons.php [hunk 2]
```diff
@@ -105,32 +104,30 @@ protected function get_views()
...
-                
     }
-        
-        
-        
-    function usort_reorder( $a, $b ) 
+
+    function usort_reorder($a, $b)
     {
         if(isset($_GET['orderby']) && isset($_GET['order']))
```
Comment (sofiya02) [inline includes/rzp-subscription-buttons.php:111]: Use and instead of &&

File: includes/rzp-subscription-buttons.php [hunk 3]
```diff
@@ -141,58 +138,57 @@ function column_title($item)
...
-        
         $count = count($payment_page);
-        for($i=0;$i<$count;$i++){
-            
-            if($i >= $offset && $i < $offset+$per_page){
+        $payment_pages = array();
+
+        for($i=0;$i<$count;$i++)
+        {
+            if($i >= $offset && $i < $offset+$per_page)
```
Comment (sofiya02) [inline includes/rzp-subscription-buttons.php:170]: Use and instead of &&,

**Review by sofiya02 — CHANGES_REQUESTED** [review]

**Comment by sofiya02** [comment]
LGTM

**Review by sofiya02 — APPROVED** [review]
LGTM

File: razorpay-sdk/libs/Requests-2.0.0/src/Transport/Curl.php [hunk 4]
```diff
@@ -0,0 +1,641 @@
...
+		$this->response_bytes      = 0;
+		$this->response_byte_limit = false;
+		if ($options['max_bytes'] !== false) {
+			$this->response_byte_limit = $options['max_bytes'];
+		}
+
+		if (isset($options['verify'])) {
+			if ($options['verify'] === false) {
+				curl_setopt($this->handle, CURLOPT_SSL_VERIFYHOST, 0);
+				curl_setopt($this->handle, CURLOPT_SSL_VERIFYPEER, 0);
```
[the remark this answers is not in the document]
Comment (chirag-patel-rp) [reply razorpay-sdk/libs/Requests-2.0.0/src/Transport/Curl.php:192]: @sonyy-m why have we disabled SSL verification here?

[the remark this answers is not in the document]
Comment (sonyy-m) [reply razorpay-sdk/libs/Requests-2.0.0/src/Transport/Curl.php:192]: @chirag-patel-rp It is from external requests library [https://github.com/WordPress/Requests](https://github.com/WordPress/Requests)

**Pull request merged**